docs/alerting/security.md

```diff
@@ -0,0 +1,79 @@
+
+`jdoe` creates a monitor that sends an email to management whenever the number of returns across across all three indices exceeds 40 per hour.
+
+Later, the user `psantos` wants to edit the monitor to run every two hours, but `psantos` only has access to `store1-returns`. To make the change, `psantos` has two options:
+
+- Update the monitor so that it only checks `store1-returns`.
+- Ask an administrator for read access to the other two indices.
+
+After making the change, the monitor now runs with the same permissions as `psantos`, including any [document-level security](../../security/access-control/document-level-security/) queries, [excluded fields](../../security/access-control/field-level-security/), and [masked fields](../../security/access-control/field-masking/). If you use an extraction query to define your monitor, use the **Run** button to ensure that the response includes the fields you need.
+
+
+## (Advanced) Limit access by backend role
+
+Out of the box, the alerting plugin has no concept of ownership. For example, if you have the `cluster:admin/opendistro/alerting/monitor/write` permission, you can edit *all* monitors, regardless of whether or not you created them. If a small number of trusted users manage your monitors and destinations, this lack of ownership generally isn't a problem. A larger organization might need to segment access by backend role.
+
+First, make sure that your users have the appropriate [backend roles](../../security/access-control/). Backend roles usually come from an [LDAP server](../../security/configuration/ldap/) or [SAML provider](../../security/configuration/saml/), but if you use the internal user database, you can use the REST API to [add them manually](../../security/access-control/api/#create-user).
+
+Next, enable the following setting:
+
+```json
+PUT _cluster/settings
+{
+  "transient": {
+    "opendistro.alerting.filter_by_backend_roles": "true"
+  }
+}
+```
+
+Now when users view alerting resources in Kibana (or make REST API calls), they only see monitors and destinations created by users who share *at least one* backend role. For example, consider three users: `jdoe`, `jroe`, and `psantos`, all of whom have full access to alerting.
+
+`jdoe` and `jroe` are on the same team at work and both have the `analyst` backend role. `psantos` has the `human-resources` backend role.
+
+If `jdoe` creates a monitor, `jroe` can see and modify it. `psantos` can't. If that monitor generates an alert, the situation is the same: `jroe` can see and acknowledge it, but `psantos` can't. If `psantos` creates a destination---you guessed it---neither `jdoe` nor `jroe` can see or modify it.
+
+
+## (Advanced) Limit access by individual
```

Comment:
Currently, its not possible to do DLS for `system-indices`

Reply:
Gotcha, pulled that from the test doc. It's commented out now in case we add it in the future.